superset-frontend/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -43,8 +43,8 @@ export const CheckboxGallery = () =>
   ));
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
Can we remove that `eslint-disable` now?

Reply:
Of course, fixed.